src/pages/store/index.vue

```diff
@@ -767,6 +931,28 @@ export default {
     selectedLanguageFilter() {
       this.scrollToTop();
     },
+    async searchQuery(newQuery) {
+      if (newQuery) {
+        try {
+          this.isSearchLoading = true;
+          this.$router.push({ query: { q: newQuery } });
+          await this.fetchSearchItems(newQuery);
+        } catch (error) {
+          this.searchItems = [];
+          // eslint-disable-next-line no-console
+          console.error(error);
+        } finally {
+          this.isSearchLoading = false;
+        }
+      } else {
+        this.$router.push({ query: {} });
```

Comment:
```suggestion
        this.$router.replace({ query: {} });
```

Reply:
[🎨 Use router replace to store query](https://github.com/likecoin/liker-land/pull/1906/commits/840dee448f133152cbd3703f4909a3926478e56e)